molo/core/tests/test_commands.py

```diff
@@ -110,35 +115,25 @@ def test_add_translated_pages_to_pages(self):
         # run command to add languages to these pages
         call_command('add_language_to_pages')
 
-        # remove the langauge from one page to ensure script works
-        # as intended for pages without a language
-        # which is to not add them as a translated page
-
-        french_article.language = None
-        french_article.save()
-
-        self.assertIsNone(french_article.language)
         # run command to add list of translated_pages to these pages
         call_command('add_translated_pages_to_pages')
 
         # test translated pages for articles
         self.assertTrue(english_article.translated_pages.filter(
             pk=spanish_article.pk).exists())
-        self.assertFalse(english_article.translated_pages.filter(
+        self.assertTrue(english_article.translated_pages.filter(
             pk=french_article.pk).exists())
         self.assertFalse(english_article.translated_pages.filter(
             pk=english_article.pk).exists())
-        self.assertFalse(english_article.translated_pages.filter(
-            pk=english_article.pk).exists())
```

Comment:
why is this check no longer valid?

Reply:
We add language to the article if it doesn't have one in the model's save method. and we add a language when we create an article in the base. So we can't really delete a page's language, which is a good thing. But not sure how to test that check

Reply:
?? but that's not what we are checking here is it?
This line was to confirm that an article isn't added to it's own `translated_pages` when the management command is run

Reply:
@KaitCrawford oh sorry got confused. Yes that is correct. The diff makes it look like it is gone, but if you look at the file the line is still there. Not sure why it does this. I checked on the file on github for this Pr and locally and it is still there

Reply:
:thinking:

Reply:
Oh, I see the line above it is exactly the same. Maybe it was accidentally duplicated in a previous PR? oh well.